ruby/lib/svix/message_api.rb

```diff
@@ -22,4 +22,33 @@ def expunge_content(app_id, msg_id)
       return @api.v1_message_expunge_content(app_id, msg_id)
     end
   end
+
+  # Creates a [`MessageIn`] with the payload already being serialized.
+  #
+  # The payload is not normalized on the server (usually whitespace outside
+  # of string literals, unnecessarily escaped characters in string and such
+  # are fixed up by the server), and is not even required to be JSON.
+  #
+  # `attributes[:payload]` must be a string. An extra attribute `content_type`
+  # is accepted that sets the `content-type` header of the payload, overwriting
+  # the default of `application/json` if specified. Other than that, the
+  # attributes are forwarded [`MessageIn.new`], so all the same ones are
+  # accepted.
+  def message_in_raw(attributes = {})
+    if !attributes.key?(:transformations_params)
```

Comment:
It's been a minute since I've used ruby, but I think you can do

```ruby
attributes[:transformations_params] ||= {}
```

to avoid the if statement if you want.

Reply:
Thanks! Looks like it works.